tests/test_files/calculator/calculator.py

```diff
@@ -1,5 +1,24 @@
-# TODO: Implement a calculator program, that can add, subtract, multiply and divide two numbers.
+# Implement a calculator program, that can add, subtract, multiply and divide two numbers.
 #  - Numbers and operators given from the user as the command line arguments.
 #  - Consider using argpasre library.
 #  - Consider using eval builtin function.
 #  - Print or log the result.
+import argparse
+
+def check_number(value):
```

Comment:
If you set the type param in argparse, it will check the type of the given argument, and the interpreter will indicate that.